Assets/Scripts/PlayerController.cs

```diff
@@ -7,7 +7,21 @@ public class PlayerController : MonoBehaviour
 
     public const string HORIZONTAL = "Horizontal";
     public const string VERTICAL = "Vertical";
-    public const string FIRE_BUTTON = "Fire1";
+    public const string FIRE_BUTTON = "Fire";
+    public const string BOMB_BUTTON = "Bomb";
+    public const float BOMB_Y_OFFSET = -1;
+
+    private static PlayerController _instance;
+    public static PlayerController instance
+    {
+        get
+        {
+            if (_instance == null)
+                _instance = FindObjectOfType<PlayerController>();
```

Comment:
This is not how you are supposed to do a singleton. Look at [http://wiki.unity3d.com/index.php/Singleton](http://wiki.unity3d.com/index.php/Singleton)